client/components/web-preview/toolbar.jsx

```diff
@@ -181,7 +182,7 @@ class PreviewToolbar extends Component {
 						<Button
 							primary
 							className="web-preview__external"
-							href={ externalUrl || previewUrl }
+							href={ logmeinUrl( externalUrl ) || previewUrl }
```

Comment:
We should probably do this change for only WPCOM sites since Jetpack sites won't recognize this param. What do you think?

Reply:
Good point, do you think we can leverage the mysites data for this?

Reply:
We can probably just use this selector:
https://github.com/Automattic/wp-calypso/blob/trunk/client/state/selectors/is-site-wpcom.js